chapter_linear-networks/linear-regression_vn.md

```diff
@@ -126,13 +126,14 @@ We will often find it convenient to refer to our entire dataset via the *design
 Here, $X$ contains one row for every example and one column for every feature.
 -->
 
-*dịch đoạn phía trên*
+Ở đây, vector $\mathbf{x}$ tương ứng với một điểm dữ liệu.
+Chúng ta sẽ thấy rằng việc truy cập đến toàn bộ tập dữ liệu thường sẽ tiện hơn thông qua việc sử dụng *ma trận* $X$.
```

Comment:
```suggestion
Chúng ta sẽ thấy rằng việc truy cập đến toàn bộ tập dữ liệu thường sẽ tiện hơn thông qua việc sử dụng *ma trận* $\mathbf{X}$.
```

Đoạn này tác giả ký hiệu nhầm `$X$`, vì đây là một ma trận nên phải in đậm `$\mathbf{X}$` như trong [bảng ký hiệu](https://github.com/aivivn/d2l-vn/blame/master/chapter_notation/index_vn.md#L26)